### ISSUE TITLE
RSS: ValueError: Required fields not set

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#2314]
We see periodic HTTP 500's in the logs from users requesting RSS URLs like this:

https://bodhi.fedoraproject.org/rss/comments/?updates=FEDORA-2017-12e76e8364

```
2018-04-24 14:34:22,567 ERROR [bodhi.server][MainThread] Error caught.  Handling HTML response.
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/pyramid/tweens.py", line 39, in excview_tween
    response = handler(request)
  File "/usr/lib/python2.7/site-packages/pyramid/router.py", line 156, in handle_request
    view_name
  File "/usr/lib/python2.7/site-packages/pyramid/view.py", line 617, in _call_view
    response = view_callable(context, request)
  File "/usr/lib/python2.7/site-packages/pyramid/config/views.py", line 181, in __call__
    return view(context, request)
  File "/usr/lib/python2.7/site-packages/pyramid/viewderivers.py", line 389, in attr_view
    return view(context, request)
  File "/usr/lib/python2.7/site-packages/pyramid/viewderivers.py", line 367, in predicate_wrapper
    return view(context, request)
  File "/usr/lib/python2.7/site-packages/pyramid/viewderivers.py", line 461, in rendered_view
    request, result, view_inst, context)
  File "/usr/lib/python2.7/site-packages/pyramid/renderers.py", line 435, in render_view
    return self.render_to_response(response, system, request=request)
  File "/usr/lib/python2.7/site-packages/pyramid/renderers.py", line 458, in render_to_response
    result = self.render(value, system_values, request=request)
  File "/usr/lib/python2.7/site-packages/pyramid/renderers.py", line 454, in render
    result = renderer(value, system_values)
  File "/usr/lib/python2.7/site-packages/bodhi/server/renderers.py", line 111, in render
    return feed.rss_str()
  File "/usr/lib/python2.7/site-packages/feedgen/feed.py", line 398, in rss_str
    feed, doc = self._create_rss(extensions=extensions)
  File "/usr/lib/python2.7/site-packages/feedgen/feed.py", line 374, in _create_rss
    item = entry.rss_entry()
  File "/usr/lib/python2.7/site-packages/feedgen/entry.py", line 186, in rss_entry
    raise ValueError('Required fields not set')
ValueError: Required fields not set
2018-04-24 14:34:22,677 ERROR [bodhi.server][MainThread] Unhandled exception raised:  <html_handler at 0x7f7d360a5280 500 Internal Server Error>
```

**Comment by bowlofeggs** [comment]
A fix for this issue is planned to be included in the upcoming 3.9.0 release: #2477.

**Comment by bowlofeggs** [comment]
The patch for this issue was deployed to Fedora's production Bodhi instance yesterday.
